lib/controller/ProfielController.class.php

```diff
@@ -202,7 +202,7 @@ public function stamboom($uid = null) {
 
 	public function verjaardagen() {
 		require_once 'view/VerjaardagenView.class.php';
-		$body = new VerjaardagenView('alleverjaardagen');
+		$body = new AlleVerjaardagenView(null);

```

Comment:
Nee, het hele idee van het verplicht maken van `$model` is om ervoor te zorgen dat bij implementatie ook een model moet worden meegegeven. Om op die manier de SOLID principes een klein beetje te forceren. Het feit dat `KomendeVerjaardagView` en `AlleVerjaardagenView` db calls doen is iets wat breekt met SOLID.

Een goede richtlijn bij de stek vind ik: Als je iets lelijks gaat doen, doe het dan waar iedereen het zien. Dus niet diep in een template.